pilot/pkg/networking/core/v1alpha3/listener_inbound.go

```diff
@@ -826,9 +833,14 @@ func (lb *ListenerBuilder) buildInboundNetworkFilters(fcc inboundChainConfig) []
 		}
 		filters = append(filters, buildMetadataExchangeNetworkFilters(istionetworking.ListenerClassSidecarInbound)...)
 	}
+	wasm := lb.push.WasmPluginsByListenerInfo(lb.node, model.WasmPluginListenerInfo{
+		Port:  fcc.port.Port,
+		Class: istionetworking.ListenerClassSidecarInbound,
+	}, model.WasmPluginTypeNetwork)
 	filters = append(filters, lb.authzCustomBuilder.BuildTCP()...)
 	filters = append(filters, lb.authzBuilder.BuildTCP()...)
 	filters = append(filters, buildMetricsNetworkFilters(lb.push, lb.node, istionetworking.ListenerClassSidecarInbound)...)
+	filters = append(filters, extension.BuildNetworkWasmFilters(wasm)...)
```

Comment:
respect the plugin phase as well

Reply:
+1, please follow the HTTP sequence. +2 if you can document it somewhere :)